client-js/gulpfile.js

```diff
@@ -148,6 +148,8 @@ gulp.task('images', function () {
 // Copy All Files At The Root Level (app)
 gulp.task('copy', function () {
   var app = gulp.src([
+    '.tmp/**/*.js',
+    '.tmp/**/*.js.map',
```

Comment:
this task copies all the files that are part of the final built app into the `dist/` directory to create a distributable version (e.g. just a bunch of static files). The task that we inherited from the polymer starter kit was just copying scripts directly from `app/`, but we added typescript compilation which produces additional javascript files (and source maps) in the `.tmp/` folder. This just makes sure those also get included in the dist package.